vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java

```diff
@@ -658,9 +666,76 @@ private void fireValidationStatusChangeEvent() {
      * @return <code>true</code> if the input element's value is populated,
      *         <code>false</code> otherwise
      */
-    @Synchronize(property = "_hasInputValue", value = "has-input-value-changed")
     protected boolean isInputValuePresent() {
```

Comment:
This getter was made protected only because we needed it to be accessible from DateTimePicker. This PR introduces a change: the getter no longer updates on `has-input-value-changed`. Do you think it's worth mentioning or even restoring the previous behavior?

Reply:
Should be fine IMO.